index.js

```diff
@@ -0,0 +1,47 @@
+const debug = require('debug')('systemic:slack');
+const { createReadStream, existsSync } = require('fs');
+const { WebClient } = require('@slack/web-api');
+
+module.exports = () => {
+  let slackClient;
+  const start = async ({ config }) => {
+    const token = process.env.SLACK_TOKEN;
```

Comment:
token should come from config, env vars should be resolved by the config component

Reply:
True, updated on last commit